Docs status updates - Fixes #150 (#268)
nautobot/nautobot: nautobot/docs/models/dcim/powerfeed.md

A power feed represents the distribution of power from a power panel to a particular device, typically a power distribution unit (PDU). The power pot (inlet) on a device can be connected via a cable to a power feed. A power feed may optionally be assigned to a rack to allow more easily tracking the distribution of power among racks.

> Each power feed is assigned an operational type (primary or redundant) and one of the following [`statuses`](https://nautobot.readthedocs.io/en/latest/models/extras/status/):

Reviewer: ```suggestion
Each power feed is assigned an operational type (primary or redundant) and a [`status`](https://nautobot.readthedocs.io/en/latest/models/extras/status/). The following statuses are available by default:
```

* Offline
* Active
* Planned
* Failed

> Status fields can now be customized via the new [`status`](https://nautobot.readthedocs.io/en/latest/models/extras/status/) model.

Reviewer: Same: IMO just remove this since it's redundant with the link above it.
